idx/memory/tag_query_test.go

```diff
@@ -559,6 +559,108 @@ func TestExpressionParsing(t *testing.T) {
 	}
 }
 
+func TestSpecialCharactersInTags(t *testing.T) {
+	type testCase struct {
+		id         schema.MKey
+		lastUpdate int64
+		tags       []string
+	}
+
+	ids = []schema.MKey{
+		test.GetMKey(0),
+		test.GetMKey(1),
+		test.GetMKey(2),
+		test.GetMKey(3),
+		test.GetMKey(4),
+		test.GetMKey(5),
+		test.GetMKey(6),
+	}
+
+	data := []testCase{}
+
+	for i := 0; i < 6; i++ {
+		data = append(data, testCase{ids[i], 1, []string{}})
```

Comment:
what is the purpose of setting these very low lastUpdate values?
seems it has nothing to do with what is being tested, and i would emit this from the testCase types

Reply:
also, having all these predeclared id's complicates things needlessly. just generate them as you need them